index.js

```diff
@@ -20,10 +20,15 @@ class AbortError extends Error {
 }
 
 module.exports = (input, opts) => new Promise((resolve, reject) => {
+	opts = Object.assign({
+		onFailedAttempt: () => {},
+		retries: 10
```

Comment:
This is already the default of `retry`, why do we need bookkeeping here? If a user cares about "attempts remaining", presumably they're capable of setting an explicit value for `retries` and doing the math themselves.

Reply:
I understand your point, however the proposal for this feature discussed in #5 includes the computation of the number of attempts left. Concerning the default for `retries`, I'm aware that `retry` already sets it. This is just a safeguard to ensure the math for `attemptsLeft` remains correct even if the default value for `retries` changes in the future for whatever reason.